content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -24,37 +24,61 @@ Here's a summary of the steps involved:
 
 **Note:** the minimum version of Microk8s needed to enable Kubeflow is 1.18
 
-### 1. Install and set up Microk8s
+### 1. Install Microk8s
 
-Run the following commands to install and setup MicroK8s:
+Single-command install MicroK8s snap:
 
 ```
-snap install microk8s --classic
+sudo snap install microk8s --classic
+```
+Confirm install:
+```
 microk8s.status --wait-ready
-# Enable common services:
+```
+
+### 2. Enable Microk8s services:
+
+Enable common services:
+```
 microk8s.enable dns dashboard storage
-# If you have a GPU, run: `microk8s.enable gpu`
 ```
+If you would like to enable GPU passthrough (optional), run: `microk8s.enable gpu`
 
-### 2. Enable Kubeflow
+### 3. Enable Kubeflow:
 
 Run the following command to enable Kubeflow:
 
 ```
 microk8s.enable kubeflow
 ```
 
-This script will print out the port number for Ambassador and for Jupyter notebook 
-servers.
+The deployment process may take a few minutes. Once completed, the script will print out the port number and credentials to access the Kubeflow Dashboard.
+
+
+## Access Kubeflow Dashboard
```

Comment:
The central dashboard is mostly referred to with lower case letters. Also, make sure you capitalize only the first letter of each heading within the page, unless there are special names involved.

Reply:
Two lines separate the previous paragraph and the heading. Maybe remove line 56.